scalableminds/webknossos
# Add task configurations download as csv

This PR adds the download one or multiple task configurations as csv to wk. The download is available 
- when creating one or multiple tasks
- in the task list view for each individual task
- at the top of of the table in the task list view to download the configurations of all loaded and filtered tasks

### URL of deployed dev instance (used for testing):
- https://addtasksettingsascsvdownload.webknossos.xyz

### Steps to test:
- Open the single task creation form and create a task. Upon success, a download of a CSV containing the configurations of the created task should be triggered,
- The same goes for the bulk creation. Please test whether the download works correctly for multiple tasks at the same time.
-  In the task list view please also check the individual configurations download and the download of all tasks shown in the table.

### Issues:
- fixes #4407

------
